core-primitives/stf-executor/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "itp-stf-executor"
+version = "0.8.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2018"
+resolver = "2"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = ["std"]
+std = [
+    "ita-stf/std",
```

Comment:
This sucks big time, but I think we will have to live with this until we abstract TrustedCalls, TrustedOp, TrustedGetter into core-primitives.

Reply:
We have the `ita-stf` dependency everywhere because of it, yes. I've looked at it a couple of times, and I'm not sure there is a 'simple' solution, provided we don't break with @brenzi 's intention to keep all the domain specific stuff in that crate (and `TrustedCall`, `TrustedGetter` etc. contain exactly the domain specific logic)